tools/logger/src/main.rs

```diff
@@ -47,26 +51,27 @@ fn main() {
                         msg.msg.unwrap(),
                     };
                 }
-                Event::Conn(c) => {
+                Event::Conn(c) if args.filter == "all" || args.filter == "conn" => {
                     log::info! {
                         "# CONN {}", c.event.unwrap()
                     };
                 }
-                Event::Addrman(a) => {
+                Event::Addrman(a) if args.filter == "all" || args.filter == "addrman" => {
                     log::info! {
                         "@Addrman {}", a.event.unwrap()
                     };
                 }
-                Event::Mempool(m) => {
+                Event::Mempool(m) if args.filter == "all" || args.filter == "mempool" => {
                     log::info! {
                         "$Mempool {}", m.event.unwrap()
                     };
                 }
-                Event::Validation(v) => {
+                Event::Validation(v) if args.filter == "all" || args.filter == "validation" => {
                     log::info! {
                         "+Validation {}", v.event.unwrap()
                     };
                 }
+                _ => {}
```

Comment:
Previously, the compiler would error if we forgot to add a new event to the logger. With this change, this will be silently ignored. I think it might be better to check inside the match like e.g. `=> { if args.filter == "all" || args.filter == "validation" { log::info!(...) } }`.